src/snowflake/snowpark/dataframe.py

```diff
@@ -5635,7 +5640,10 @@ def describe(
                     # for string columns, we need to convert all stats to string
                     # such that they can be fitted into one column
                     if isinstance(t, StringType):
-                        if name in ["mean", "stddev"]:
+                        if not strings_include_math_stats and name in [
+                            "mean",
+                            "stddev",
+                        ]:
                             agg_cols.append(to_char(func(lit(None))).as_(c))
                         else:
                             agg_cols.append(to_char(func(c)))
```

Comment:
```suggestion
                        if strings_include_math_stats and name in [
                            "mean",
                            "stddev",
                        ]:
                            agg_cols.append(to_char(func(c)))
                        else:
                            agg_cols.append(to_char(func(lit(None))).as_(c))
```
may I suggest we re-write it this way to make it more readable. the `not strings_include_math_stats` makes me do extra compute

Reply:
Absolutely 👍